lib/tiled_component.dart

```diff
@@ -19,14 +19,19 @@ class TiledComponent extends Component {
 
   @override
   void render(Canvas canvas) {
-    _tiled.render(canvas);
+    //_tiled!.generate();
+    _tiled!.render(canvas);
   }
 
   @override
-  bool loaded() => _tiled.loaded();
+  Future<void> onLoad() async {
+    await future;
+  }
+
+  bool loaded() => _tiled!.loaded();
 
-  get future => _tiled.future;
+  get future => _tiled!.future;
```

Comment:
I don't think this getter is needed?

Reply:
Wothout the getter, ` await tiled.future;` does not work, see `tiled_test.dart`